tools/clean.bat

```diff
@@ -0,0 +1,5 @@
+make clean
+idf.py clean fullclean
+rmdir /s /q build
+rmdir /s /q spiffs_image
+mkdir spiffs_image
```

Comment:
This should be a `makefile` target, like `make fullclean`, not a batch file. That way it's cross-platform. This is what you want:

```
fullclean: clean
	idf.py fullclean
	git clean -fX
	-@rm -rf ./docs/html
	-@rm -rf ./spiffs_image/*
	$(MAKE) -C ./tools/sandbox_test clean
	$(MAKE) -C ./tools/hidapi_test clean
	$(MAKE) -C ./tools/bootload_reboot_stub clean
	$(MAKE) -C ./tools/font_maker clean
	$(MAKE) -C ./tools/swadgeterm clean
	$(MAKE) -C ./tools/reboot_into_bootloader clean
	$(MAKE) -C ./tools/spiffs_file_preprocessor clean
```